docs/sources/view-and-analyze-profile-data/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: View and analyze profile data
+menuTitle: View and analyze profile data
+description: How to use Pyroscope to view and analyze profile data.
+aliases:
+  - ../ingest-and-analyze-profile-data/
+weight: 50
+keywords:
+  - pyroscope
+  - UI
+  - performance analysis
+  - flamegraphs
+  - CLI
+---
+
+# View and analyze profile data
+
+Profiling data can be presented in a variety of formats presents such as:
+- **Flamegraphs**: Visualize call relationships and identify hotspots.
+- **Tables**: View detailed statistics for specific functions or time periods.
+- **Charts and graphs**: Analyze trends and compare performance across different metrics.
+
+## Viewing profiles
+
+Pyroscope offers both a Command Line Interface (CLI) and an Application Programming Interface (API) to interact with and retrieve profiling data. These tools provide flexibility in how you access and manage your profiling information.
+
+You can export profiling data from Pyroscope in various formats:
+- **JSON**: JSON object easy to integrate with other tools and scripts
```

Comment:
The most popular is pprof and we support that to be precised it gzip compressed pprof `foo.pprof.gz`

Reply:
Added.